src/controllers/accommodationController.js

```diff
@@ -344,4 +346,20 @@ export default class AccommodationController {
       await updateLikeHelper(res, likes, like, false, 'like', strings.likes.success.LIKED, id, req.user.payload.id);
       await updateLikeHelper(res, likes, like, true, 'unlike', strings.likes.success.DISLIKE, id, req.user.payload.id);
     }
+
+  static async viewTopRated(req, res) {
+    const accommodations = await getAllAccommodations();
+    const values = [];
+
+    accommodations.forEach(accommodation => {
+      if (!isNaN(accommodation.dataValues.averageRating)) {
+        values.push(accommodation.dataValues);
+      }
+    });
+    const sorted = values.sort((low, high) => high.averageRating - low.averageRating).slice(0, 5);
+
+    return responseUtil(res, 200, (!sorted.length)
+      ? NO_ACCOMMODATION : HIGHEST_RATED, sorted);
+
```

Comment:
Kindly make the ternary condition return `404` when there are no rated accommodations`(!sorted.length)`.
And make the message more relevant `No Rated Accommodation Found!`